tests/integration_tests.rs

```diff
@@ -2,10 +2,9 @@ use std::env;
 use std::thread;
 
 extern crate weld;
-extern crate weld_common;
 extern crate libc;
 
-use weld_common::WeldRuntimeErrno;
+use weld::weld_common::WeldRuntimeErrno;
```

Comment:
should we rename this module to `common` or just put it in the main weld `lib.rs`?

Reply:
Renamed the module to `common`